README.md

```diff
@@ -1,31 +1,23 @@
-# News Summary challenge
+# News Summary App
+This is a news summary single web page app built in JavaScript.
 
-* Feel free to use Google, your notes, books, etc. but work on your own.
-* If you refer to the solution of another coach or student, please put a link to that in your README.
-* If you have a partial solution, **still check in a partial solution** and send in a pull request.
-* You must submit a pull request to this repo with your code by 9am Monday morning.
+## Pre-requisites
+In order to run this application, please install Node.js on your local machine.
 
-## Challenge
-
-As usual please start by forking this repo.
-
-You'll create an app that summarises the news.
-
-### Guidance
+## Run locally
+In order to run this application locally, please run the following commands:
+```bash
+cd /path/to/the/root/folder/of/this/project
+npm install
+npm start
+```
```

Comment:
I followed these instructions but it doesn't seem to work - just a white blank screen with today's date and "News:" subheading